x-pack/plugins/security_solution/public/cases/components/use_all_cases_modal/index.tsx

```diff
@@ -6,13 +6,14 @@
  */
 
 import React, { useState, useCallback, useMemo } from 'react';
-import { CaseStatuses } from '../../../../../cases/common';
-import { Case, SubCase } from '../../containers/types';
+import { Case, CaseStatuses, CommentRequestAlertType, SubCase } from '../../../../../cases/common';
 import { AllCasesModal } from './all_cases_modal';
 
 export interface UseAllCasesModalProps {
-  onRowClick: (theCase?: Case | SubCase) => void;
+  alertData?: Omit<CommentRequestAlertType, 'type'>;
```

Comment:
`alertData` object and `updateCase` callback get passed to `postComment` on the `cases` side

Reply:
https://github.com/elastic/kibana/pull/95176/files#r605461275